app/src/main/java/com/vero/tasky/agenda/presentation/agendaevent/EventDetailsEvent.kt

```diff
@@ -17,7 +18,11 @@ sealed interface EventDetailsEvent {
     class ToTimeChanged(val time: LocalTime) : EventDetailsEvent
     class ReminderChanged(val reminderRange: ReminderRange) : EventDetailsEvent
     class DeletePhoto(val key: String) : EventDetailsEvent
-    class AddAttendee(val email: String) : EventDetailsEvent
+    class OnEmailUpdated(val email: String) : EventDetailsEvent
+    object AddAttendee: EventDetailsEvent
+    object HideAddAttendeeDialog: EventDetailsEvent
+    object ShowAttendeeDialog: EventDetailsEvent
```

Comment:
I agree with your opinion and prefer to do it like you. In this case, I add these events and a field to my state just because of this (screenshot), I need to hide dialog after success adding, and I thought it will be clear to do it using one variable (another option is to add new UiEvent and hide dialog from LaunchEffect, but I preferred one variable option)
![image](https://github.com/AndreVero/Tasky/assets/39765531/778d2a31-4d78-4b9c-9ca5-a76f18159080)